<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Окно выбора</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="view_certificate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть сертификат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_certificate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить новый сертификат</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_convert">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Конвертировать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
